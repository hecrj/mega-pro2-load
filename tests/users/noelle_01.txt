10
10
20
30
40
50
60
70
80
90
100

10
4 6 8 9 0 0 0 2 7 0 0 0 1 0 3 5 0 0 0

10 0
20 0
30 0
40 0
50 0
60 0
70 0
80 0
90 0
100 0

-1 5 0

-4 4
1 4
0

-1 4 10

-4 9
1 4
0

-1 4 20

-4 2
1 8
0

-4 3
1 8
0
-4 6
1 8
0

-4 3
1 8
0

-1 8 30

-4 4
0
10 1 2 3 4 5 6 7 8 9 10

-4 5
1 8
0

-1 8 50

-4 5
1 10
9 1 2 3 4 5 6 7 8 9

-1 10 100

-5 0 100

-2 101

-3 101

-6
