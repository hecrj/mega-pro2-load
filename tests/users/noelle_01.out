Peticion procesada y servidores
0 0
Peticion procesada y servidores
1 1
4
Peticion procesada y servidores
2 1
4
Peticion procesada y servidores
3 1
2 6
Peticion procesada y servidores
4 1
2 6
Peticion procesada y servidores
5 2
5
Pelicula mas solicitada
4 2
Peticiones pendientes
5 10 100 102
Servidores ocupados
5 5 1
